spec/unit/provider/systemd_unit_spec.rb

```diff
@@ -68,6 +68,10 @@
     double("shell_out", exitstatus: 0, error?: false, stdout: "masked")
   end
 
+  let(:shell_out_bad) do
```

Comment:
does not look like this is being used anymore, so this can be removed.

Reply:
Looks like https://github.com/chef/chef/blob/f3e3f07ce70fa388f2eb2f95bfa22b3a7bde0c0e/spec/unit/provider/systemd_unit_spec.rb#L59 is the only one ever used, so we either do not need the rest or need to add the appropriate tests for them in this provider.